src/main/java/uk/gov/ida/rp/testrp/authentication/SessionFactory.java

```diff
@@ -96,12 +96,16 @@ public Session provide() {
             }
         }
 
+        // TODO: remove the EIDAS_PARAM completely
+        if (containsQueryParam(EIDAS_PARAM) && !journeyHint.isPresent()) {
```

Comment:
Can this now be removed?

Reply:
I can remove the TODO comment, but we should probably leave the code for now.